molo/core/templates/core/article_tags.html

```diff
@@ -2,9 +2,9 @@
 {% load wagtailcore_tags wagtailimages_tags static core_tags i18n %}
 {% block content %}
   <div class="article-teaser">
-  <h1 class="article-teaser__heading">{{ tag }}</h1>
+  <h1 class="article-teaser__heading">{{ object_list.0.title }}</h1>
```

Comment:
linked to `get_context_data` suggestion - it would be better to do `{{ tag.title}}`